docs/src/styles/docs/base.scss

```diff
@@ -79,7 +79,7 @@ body {
   line-height: 1.5;
 }
 
-a {
+a:not([class]) {
```

Comment:
Do you think there is any value in adding a comment explaining what we are doing with the `not` pseudo selector here? Thinking it might be helpful since we are applying to all `a` elements

Reply:
Yeah, I agree. Added a comment in latest commit